components.css

```diff
@@ -0,0 +1,10 @@
+/* Importing all the component to this single file */
```

Comment:
I suggest changing `component` to `components`

Reply:
done